robot_ws/src/object_tracker_robot/launch/main.launch.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this
+# software and associated documentation files (the "Software"), to deal in the Software
+# without restriction, including without limitation the rights to use, copy, modify,
+# merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+# INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+# PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+# HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+# OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import os
+import sys
+import launch
+import launch_ros.actions
+
+from launch import LaunchDescription
+from ament_index_python.packages import get_package_share_directory
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', '..', 'launch'))  # noqa
+
+
+def generate_launch_description():
+
+    #################################################
+    ##  Turtlebot Waffle PI Object Tracket Launch  ##
+    #################################################
+
+    ####################################
+    ##  Get Raspicam Params from YAML ##
+    ####################################
+#    raspicam_params = os.path.join(
```

Comment:
Don't comment out raspicam, launch it conditionally instead. Declare a launch argument so you can invoke with something like `ros2 launch object_tracker_robot main.launch.py launch_camera_node:=<off, usbcam, raspicam>` and launch the relevant node.

Reply:
Added 2 arguments run_usb_cam and run_pi_cam to select the camera and bydefault, camera node selects usb camera.

**For USB Camera (default)**, ros2 launch object_tracket_robot main.launch.py
**For Raspicam**, ros2 launch object_tracket_robot main.launch.py run_pi_cam:=True run_usb_cam:=False